### DISCUSSION TITLE
Update the docu?

### DISCUSSION THREAD

**Discussion by Stephan-4711** [StrausFuenf/hassio-addons#45] [General]
When I'm inside the documentation of this addon in the homeassistant addons.
There is a lot of refrerencing to frenck, also the add to home aisstant button in this link is linked to frenck.
I thing this should be updated shouldn't it?

hassio/addon/aa433fa6_vaultwarden/documentation

there is the link:
https://my.home-assistant.io/redirect/supervisor_addon/?addon=a0d7b954_bitwarden&repository_url=https%3A%2F%2Fgithub.com%2Fhassio-addons%2Frepository

this points to frenck

and here:
This is forked from the original Vaultwarden [community-addon](https://github.com/hassio-addons/repository/tree/master/bitwarden)

I would also make a link to your fork and not only at the buttom, so every body can find this easily

**Comment by strausfuenf** [comment]
Hey,

thanks for the input.
At this point I think thats a good Idea.
At first the Repo was just to show that It works and I hoped that Frenck will add Push Notifications himself.

Tbh I thought I already Updated the Button.. 
Will Update the Docu this week!

Thanks

**Comment by strausfuenf** [comment]
Added with #46 
Should be Visible in Homeassistant with the next Addon Update.
Since its just documentation I dont want to trigger a new Version
